guides/common/modules/proc_synchronizing-a-custom-repository.adoc

```diff
@@ -0,0 +1,31 @@
+[id="Synchronizing_a_Custom_Repository_{context}"]
+= Synchronizing a Custom Repository
+
+Use this procedure to synchronize content from a custom repository on a connected {ProjectServer} and a disconnected {ProjectServer} via {ISS} (ISS) Network Sync.
+
+Follow the procedure for the connected {ProjectServer} before completing the procedure for the disconnected {ProjectServer}.
+
+.Connected {ProjectServer}
+. In the {ProjectWebUI}, navigate to *Content* > *Products*.
+. Click on the custom product.
+. Click on the custom repository.
+. Copy the *Published At:* URL.
+* On the connected {ProjectServer}, download the file `http://{foreman-example-com}/pub/katello-server-ca.crt` using `curl` or `wget`.
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# curl [opts] https://{foreman-example-com}/pub/katello-server-ca.crt
+----
+. Continue with the procedure on disconnected {ProjectServer}.
+
+.Disconnected {ProjectServer}
+. In the {ProjectWebUI}, navigate to *Content* > *Products*.
+. Create your custom product with the following:
+* Upstream URL: Paste the link that you copied earlier.
+* Transport the file to the disconnected server using any method.
```

Comment:
If we change the instruction to download the file on the disconnected {ProjectServer} rather than the connected, we can eliminate this step of transporting it. It will already be on the disconnected server.